modules/sprint.js

```diff
@@ -0,0 +1,105 @@
+import axios from 'axios'
+import plateBarcode from '@/modules/plate_barcode'
+import config from '@/nuxt.config'
+
+const query = `mutation printRequest($printRequest: PrintRequest!, $printer: String!) {
+  print(printRequest: $printRequest, printer: $printer) {
+    jobId
+  }`
+
+const headers = {
+  headers: {
+    'Content-Type': 'application/json'
+  }
+}
+
+// Will create a new layout object for a print job
+// Requires barcode which will be used for barcode and text field
+const createLayout = (barcode) => ({
```

Comment:
> It would be good to know what printer type this layout is for. Maybe it could be pulled out to a constant/method if not config. A way that can be used in testing too might be useful as the test is currently duplicating it

I think I understand what you mean but I am not really sure that it is relevant in this case. We are only printing destination plate labels so it wil always be a plate and that is reflected in the printers available. I think it could become more relevant if we expand sprint usage in lighthouse-ui at a later date.